distribution/ltp-upstream/lite/runtest.sh

```diff
@@ -85,6 +74,72 @@ function ltp_test_build()
 	test_msg pass "LTP build/install successful"
 }
 
+function hugetlb_nr_setup()
+{
+       grep -q hugetlbfs /proc/filesystems || return
+       echo 3 >/proc/sys/vm/drop_caches
+
+       cat hugetlb.inc > hugetlb
```

Comment:
Why `cat >`, and not `cp`? Is `hugetlb` a special file in `/proc` or something? Because from this function it's not clear what the current directory is...

Reply:
I didn't think too much for choosing 'cat' or 'cp', here just make a copy of the runtest file which we can hack it according to system available memory size.
After doing that, mv the hugetlb file to LTP installed directory and run with it.

Reply:
and btw, you can find the config file at:
https://github.com/linux-test-project/ltp/blob/master/runtest/hugetlb